### DISCUSSION TITLE
Translation of a short 5 page PDF in Urdu language not showing English Translation

### DISCUSSION THREAD

**Discussion by agent-imran** [immersive-translate/immersive-translate#1873] [Help]
### 概述 | Summary

I uploaded this PDF to the PDF Pro tool for translation from Urdu in English.  The results did not give me any translation.

The source file is attached.
[tabqaat-urdu (30-34).pdf](https://github.com/user-attachments/files/16119995/tabqaat-urdu.30-34.pdf)

Result:
https://readit.plus/a/Gquti/8b8c852b-c567-46b7-8af1-1a06e19f2cfd

### 补充信息 | Additional information

Using Google Chrome

**Settings:**
Translation Service:  Gemini
Target Language: English
Mode: Bilingual Comparison

**Comment by TaoLoading** [comment]
PDF Pro currently has issues with recognizing scanned Urdu language documents. This issue has been noted and reported, and we will inform you once the functionality is optimized. Currently, PDF Pro is more suitable for translating academic papers, references, and other academic materials.
